test/defs/Dockerfile.buildargs

```diff
@@ -0,0 +1,8 @@
+FROM alpine
```

Comment:
Apologies for not thinking about this before... but we try to ensure any tests consuming stuff from Docker hub are in `e2e/Docker`... so that it's very clear where things are that could contribute to hitting rate limits etc.

Would you be able to either:

* Move anything that's building `FROM <docker hub container>` into `e2e/docker` -or-
* Modify the definition files so that they build `FROM <our local registry image` (probably requires templating to substitute the correct FROM`.